api/v1alpha1/sandbox_types.go

```diff
@@ -21,9 +21,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
`create` and `disable` are verbs and `ready` is adjective or noun... can we make it more consistently?

Reply:
Let's start with #18 (I like the design!) and if users have any feedback, we can always improve.

Reply:
done.